src/hooks/useNetworkGraph.ts

```diff
@@ -254,7 +254,8 @@ export const useNetworkGraph = ({
     renderer.current?.setSetting('nodeReducer', (node: string, data: any) => {
       const nodeAttribute = graph.current?.getNodeAttributes(node);
 
-      const labelMatches = searchValueMainLabel && node.toLowerCase().includes(searchValueMainLabel.toLowerCase());
+      const labelMatches =
```

Comment:
Would suggest `isLabelMatch` instead of `labelMatches` as variable name to meke it clear that this is a boolean value. Using `is` as a prefix is widely used in our repo as well.

Reply:
Okay, but I used `doesLabelMatch`.